test/project/config/crds/patches/creatures_kraken_patch.yaml

```diff
@@ -0,0 +1,15 @@
+# This file is generated by kubebuilder
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: krakens.creatures.testproject.org
+spec:
+  conversion:
+    strategy: Webhook
+    webhookClientConfig:
+      caBundle: XG4= # this is "\n" used as a placeholder
```

Comment:
might also want to say "otherwise it will be rejected by the api server for being blank, but we're going to set it later using a certmanager (or potentially a patch if not using certmanager)".